src/cells/actions/thunks.ts

```diff
@@ -69,7 +69,8 @@ export const deleteCellAndView = (
 export const createCellWithView = (
   dashboardID: string,
   view: NewView,
-  clonedCell?: Cell
+  clonedCell?: Cell,
+  redirect?: (destinationId: string) => void | undefined
```

Comment:
what's the ```void | undefined``` bit about? shouldn't `void` be fine? 

https://stackoverflow.com/a/58885486/92446